apps/src/codebridge/hooks/useInitialSources.ts

```diff
@@ -19,34 +19,47 @@ export const useInitialSources = (defaultSources: ProjectSources) => {
   const levelStartSource = useAppSelector(
```

Comment:
I can't comment higher, but can you update the comment above to include the new logic with template sources?